src/types/Comment.ts

```diff
@@ -1,6 +1,13 @@
 export interface Comment {
   id: number;
-  postId: number;
+  postId: number | undefined;
+  name: string;
+  email: string;
+  body: string;
+}
+
+export interface NewComment {
+  postId?: number;
```

Comment:
It is not clear why postId is optional in NewComment interface. Please double-check if this is intended or if it should be required.